### DISCUSSION TITLE
Can I disconnect DIO0 and DIO1?

### DISCUSSION THREAD

**Discussion by radio-satellites** [jgromes/RadioLib#849] [Q&A] [answered]
Hi All,

I'm doing a project with LoRa where I have an extremely limited pin count. In fact, there are only enough pins to connect the basic SPI and power functions, but not the standard DIO0 and DIO1 pins used for the end of packet transmission that are required by every example in the SX1278 folder. 

I know that you can poll the internal registers of the LoRa device to find out whether or not the packet has ended, hence eliminating the requirement for the utilization of these pins. 

Does this library support this, and if not, is it necessary to connect the DIO0 and DIO1 pins on the SX1278?

73 de VE3SVF!

**Comment by radio-satellites** [comment]
After reading some wiki pages, I realized I forgot to mention that it will only be used for the transmission (no RX) of LoRa and FSK.

**Reply by jgromes** [reply, marked as answer]
Strictly speaking, the lbirary does not support this - at least DIO0 is required for operation. However, if you only need to transmit, you can set the DIO0 pin to `RADIOLIB_NC`, and the ignore the TX timeout errors codes. The packet will be transmitted.

**Reply by radio-satellites** [reply]
Thanks @jgromes !
